core/flyout_base.ts

```diff
@@ -799,7 +799,7 @@ export abstract class Flyout extends DeleteArea implements IFlyout {
       blockInfo: toolbox.BlockInfo, gaps: number[], defaultGap: number) {
     let gap;
     if (blockInfo['gap']) {
-      gap = parseInt(blockInfo['gap'].toString());
+      gap = parseInt(String(blockInfo['gap']));
```

Comment:
d25dca438cfc02c163799f4f9e2cf3c9c718ce3c uses the string constructor instead of template litterals, but I'm not sure why. Are they semantically different?

Reply:
No difference.  We had a lovely diverse mix of:
1. `String(foo)`
2. `foo.toString()`
3. `'{$foo}'`
4. `foo + ''`

I really don't have any preferences other than getting rid of `foo.toString()` since that throws if foo is null or undefined.  This PR generally uses `'{$foo}'` for simple variables, and `String(foo.bar)` for anything more complicated (I find `'{$foo["bar"].toUpperCase()}'` to be harder to read).

If we want to standardize on something, that's the team's choice.  I just want to get rid of `.toString()` here.